chat-plugins/mafia.js

```diff
@@ -1395,7 +1395,7 @@ class MafiaTracker extends Rooms.RoomGame {
 		if (this.cohosts.includes(user.userid)) return `${targetString} a cohost.`;
 		if (!force) {
 			for (const alt of user.getAltUsers(true)) {
-				if (this.players[alt.userid]) return `${self ? `You already have` : `${user.userid} already has`} an alt in the game.`;
+				if (this.played.includes(alt.userid)) return `${self ? `You already have` : `${user.userid} already has`} an alt in the game.`;
```

Comment:
`an alt that has played in the game.` since were using played and not current players now.